src/ui/pages/side-menu.page.ts

```diff
@@ -0,0 +1,27 @@
+import { Locator, Page } from "@playwright/test";
+import { SideMenuItem } from "types/side-menu.types";
+
+export class SideMenuComponent {
+    readonly salesPortalButton: Locator;
+    readonly userDropdown: Locator;
+    readonly signOutButton: Locator;
+    constructor(protected page: Page) {
+        this.salesPortalButton = this.page.locator("span.fs-4");
+        this.userDropdown = this.page.locator("#dropdownUser1");
+        this.signOutButton = this.page.locator("#signOut");
+    }
```

Comment:
И так можно для каждого класса которому нужна `Page`
```suggestion
export class PageHolder {
  constructor(page: Page) {}
}
export class SideMenuComponent extends PageHolder {
    private readonly salesPortalButton = this.page.locator("span.fs-4");
    private readonly userDropdown = this.page.locator("#dropdownUser1");
    private readonly signOutButton = this.page.locator("#signOut");
```